mods/tuxemon/l18n/README.md

```diff
@@ -0,0 +1,7 @@
+Localization
+===========
+
+All translations are generated using [Weblate](https://hosted.weblate.org/projects/tuxemon/).
```

Comment:
Refer back to the main section.

Reply:
Still needs to be done.

Reply:
Not sure I understand. Which main section? Do you mean the CONTRIBUTING doc?

Reply:
Yeah

Reply:
Makes sense. I added a markdown hypertext reference.